staging/src/k8s.io/client-go/plugin/pkg/client/auth/exec/exec.go

```diff
@@ -152,6 +192,12 @@ type Authenticator struct {
 	group schema.GroupVersion
 	env   []string
 
+	// Used to avoid log spew by rate limiting install hint printing. We didn't do
+	// this by interval based rate limiting alone since that way may have prevented
+	// the install hint from showing up for kubectl users.
+	*sometimes
```

Comment:
Why is this embedded?

Reply:
Just for readability. I'll use `sometimes *sometimes` instead.

Reply:
Embed means Authenticator will have a public Do() method which doesn't make sense.

Reply:
Whoops. I got my go-isms mixed up. Make sense.

Reply:
ae979674f2a.